db/migrate/20230908082813_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
+      t.integer :posts_counter, default: 0
+
+      t.timestamps
+    end
```

Comment:
- [ ] Kindly fix the error in the migration files, you have more than one migration file with the name CreateUser.


![tests](https://github.com/kayjeee/Hi-Blog_app_kayjeee/assets/53124548/f3756546-94ea-461d-89b9-168efc00fe4d)